android/app/build.gradle

```diff
@@ -132,8 +129,6 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-image-picker')
```

Comment:
Yes I know. That command is stupid and don't understand that changes were made to the files. So you can need to stage carefully. 
That said there is a new replacement called "react-native-upgrafe-git" or something like that. This looks into history and is much better for such situations.